src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -123,7 +123,7 @@
  
 
   <Import Project="$(CSharpTargetsFile)"
-          Condition="'$(TargetFrameworkIdentifier)' != '.NETPortable' and '$(MSBuildProjectExtension)' == '.csproj'" />
+          Condition="'$(TargetFrameworkIdentifier)' != '.NETPortable' and '$(MSBuildProjectExtension)' == '.csproj' and '$(UsingMicrosoftNETSdk)' != 'true'" />
```

Comment:
Just wondering, do we have any project we build that targets .NETPortable? In case we don't then it seems like the above and bellow imports are dead code. If we do have .NETPortable projs, wouldn't we want to add the condition `'$(UsingMicrosoftNETSdk)' != 'true'`

Reply:
I'm not sure, and I'm not completely sure how to check. Do you or @weshaggard know?  I can remove all the .NETPortable stuff, if we are no longer using it.

AFAIK, the NET.Sdk doesn't support .NETPortable at all, which is why this PR was made, but never merged: https://github.com/dotnet/sdk/pull/889.  So if we still still need .NETPortable, we probably can't use the NET.Sdk without doing some work ourselves.

Reply:
My goal will be to eventually clean out most if not all of this targets file. Lets keep that separate from the initial switching to sdk projects though.